docs/int/openapi-v3.yaml

```diff
@@ -728,7 +728,9 @@ paths:
               schema:
                 type: array
                 items:
-                  type: boolean
+                  type: array
```

Comment:
@manuel-delvillar This OpenAPI spec change does not reflect the new `similarities/results` endpoint. Please update this file and ensure the api-tests are running correctly.